trade-finance-manager-api/src/v1/graphql-mappings/mappings/facilities/mapFacilityStage.js

```diff
@@ -0,0 +1,25 @@
+const CONSTANTS = require('../../../../constants');
+
+const mapFacilityStage = (facilityStage) => {
+  let mapped;
+
+  // NOTE: we currently have two different data sources.
+  // - Bond & Loan facilities have facilityStage string of 'issued' or 'unissued'.
+  // - Cash & Contingent facilities have hasBeenIssued boolean.
```

Comment:
I feel we could reduce some of these comments by assigning our or checks to constants?

Reply:
As these are existing and due to the number of changes in this PR, we shall leave this as is